templates.go

```diff
@@ -167,19 +168,28 @@ func NewCVPage(co *Company, in *CV) *Page {
 	)
 }
 
-func experienceYears(projects []Project) int {
+func experienceYears(projects []Project) string {
 	// calculate number of hears
 	var from int
 	var to int
 	for _, p := range projects {
-		if from == 0 || p.Period.FromYear < from {
-			from = p.Period.FromYear
+		if FromYear, err := strconv.Atoi(p.Period.FromYear); err == nil {
+			if from == 0 || FromYear < from {
+				from = FromYear
+			}
 		}
-		if p.Period.ToYear > to {
-			to = p.Period.ToYear
+		if ToYear, err := strconv.Atoi(p.Period.ToYear); err == nil {
+			if ToYear > to {
+				to = ToYear
+			}
+		} else if p.Period.ToYear == "present" || p.Period.ToYear == "current" {
+			to = time.Now().Year()
 		}
 	}
```

Comment:
I guess you would only really need to add

```suggestion
if to == 0 {
	to = time.Now().Year()
}
```

Reply:
I'll check, and yes it is a valid solution.